### PR TITLE
Copilot Chat: Adding missing appsettings chatparticipant config to bicep template

### PR DESCRIPTION
**Pull request by teresaqhoang**
### Motivation and Context

Adding `ChatStore:Cosmos:ChatParticipantsContainer` config setting to the bicep template for the web app.
In response to: https://github.com/orgs/microsoft/projects/852/views/5?pane=issue&itemId=32088282

Did a test deployment and verified that new setting is in the config! 
 {

    "name": "ChatStore:Cosmos:ChatParticipantsContainer",
    "value": "chatparticipants",
    "slotSetting": false

  },
  
image

Then tested deployed webapi with local webapp instance and chatbot is running as expected
  
### Contribution Checklist

- [x] The code builds clean without any errors or warnings
- [x] The PR follows SK Contribution Guidelines (https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md)
- [x] The code follows the .NET coding conventions (https://learn.microsoft.com/dotnet/csharp/fundamentals/coding-style/coding-conventions) verified with `dotnet format`
- [x] All unit tests pass, and I have added new tests where possible
- [x] I didn't break anyone :smile:

### CODE REVIEW

File: samples/apps/copilot-chat-app/deploy/main.bicep [hunk 1]
```diff
@@ -208,6 +208,10 @@ resource appServiceWebConfig 'Microsoft.Web/sites/config@2022-09-01' = {
         name: 'ChatStore:Cosmos:ChatMemorySourcesContainer'
         value: 'chatmemorysources'
       }
+      {
```
Comment (glahaye) [inline samples/apps/copilot-chat-app/deploy/main.bicep:211]: Please update the corresponding JSON ARM template too using the following command:
az bicep build -f .\main.bicep

Let me know if you want me to give you the JSON file instead.

The JSON file is needed by the "Deploy to Azure" button

**Review by glahaye — APPROVED** [review]

File: samples/apps/copilot-chat-app/deploy/main.json [hunk 2]
```diff
@@ -4,8 +4,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.17.1.54307",
-      "templateHash": "14923066769528474387"
+      "version": "0.16.2.56959",
+      "templateHash": "18037485528098010448"
```
Comment (gitri-ms) [inline samples/apps/copilot-chat-app/deploy/main.json:8]: Is this version downgrade intentional?

Comment (glahaye) [reply samples/apps/copilot-chat-app/deploy/main.json:8]: It makes no difference.  It just lists the version of the Bicep to ARM JSON compiler used to generate this template.

As an aside, one can upgrade their bicep tool with this command:
az bicep upgrade

Comment (gitri-ms) [reply samples/apps/copilot-chat-app/deploy/main.json:8]: Cool thx

**Pull request merged**